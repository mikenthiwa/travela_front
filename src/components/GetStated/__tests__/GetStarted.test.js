import React from 'react';
import GetStarted from '..';


describe('<GetStarted />', () => {
  const wrapper = shallow(<GetStarted />);

  it('should render without crashing', () => {
    const wrapper = shallow(<GetStarted />);
    expect(wrapper).toMatchSnapshot();
  });
});
