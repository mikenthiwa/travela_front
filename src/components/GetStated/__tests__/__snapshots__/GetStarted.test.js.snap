// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GetStarted /> should render without crashing 1`] = `
<div
  className="started"
>
  <div
    className="card-layout card-layout--start"
  >
    <img
      alt=""
      className="photo"
      src="home.png"
    />
    <div
      className="details"
    >
      <p
        className="ready"
      >
        Get Travel Ready
      </p>
      <p
        className="confirm"
      >
        Upload details of your travel documents today and avoid last minute hassle.
      </p>
      <Link
        replace={false}
        to="/travel_readiness"
      >
        Get Started
      </Link>
    </div>
  </div>
</div>
`;
